.github/workflows/ci.yaml

```diff
@@ -156,6 +156,9 @@ jobs:
         echo "##[group]List workload"
           tanzu apps workload list
         echo "##[endgroup]"
+        echo "##[group]List workload with output json"
+          tanzu apps workload list --output json
```

Comment:
nit: This is good to validate in CI. What do you think about adding a `jsonpath` validation from the output?

Example: ` -o jsonpath='{.status.conditions[?(@.type == "Ready")].status'})`

Reply:
there is no status blocks so I have updated to check for workload with the name `petclinic`